asv.conf.json

```diff
@@ -0,0 +1,18 @@
+{
+  "environment_type": "conda",
+  "matrix": {
+    "ipython": [],
+    "matplotlib": [],
+    "nose": [],
+    "numpy": [],
+    "pandas": [],
+    "scipy": []
+  },
+  "project": "scikit-bio",
+  "project_url": "http://scikit-bio.org/",
+  "pythons": [
+    "2.7",
+    "3.4"
```

Comment:
Don't see why not. Note that it will increase the benchmark run time by 33%. Although I think I'm going to set an upper limit on the number of commits to run against (asv can automatically downsample) in order to have more predictable and reasonable result feedback times. See [this issue](https://github.com/anderspitman/autobencher/issues/11)